Tests/Functional/Action/Launch/LaunchActionTest.php

```diff
@@ -44,6 +45,15 @@ protected function setUp(): void
         $this->client = static::createClient();
     }
 
+    protected function tearDown(): void
+    {
+        if (Carbon::hasTestNow()) {
+            Carbon::setTestNow(new DateTime());
```

Comment:
I think you don't need to provide any DateTime, just clear the test instance
```
     * To clear the test instance call this method using the default
     * parameter of null.
     *
     * /!\ Use this method for unit tests only.
     *
     * @param Closure|static|string|null $testNow real or mock Carbon instance
     */
    public static function setTestNow($testNow = null);
```

Reply:
true, less explicit, but works as well.

fixed :)